test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -176,6 +176,26 @@ class test_ucp_sockaddr : public ucp_test {
         return skip;
     }
 
+    bool is_rdmacm_supported()
+    {
+        auto e                = create_entity(); // temporary entity
+        ucp_context_h context = e->ucph();
+        bool result           = false;
+
+        for (ucp_rsc_index_t cm_idx = 0; cm_idx < context->config.num_cm_cmpts;
+             ++cm_idx) {
+            std::string cm_name = ucp_context_cm_name(context, cm_idx);
+            if (cm_name.compare("rdmacm") == 0) {
+                result = true;
+                goto out;
+            }
+        }
```

Comment:
can we make this test less intrusive, without so much looking into UCP internals?

Reply:
we need to know whether RDMACM is supported or not to set `UCX_NET_DEVICES` to either IB device name (to create auxiliary IB transport) or to IP interface name (to create auxiliary TCP transport).

Reply:
what if we set to both?